playerctl/playerctl-cli.c

```diff
@@ -372,19 +372,43 @@ int main (int argc, char *argv[])
     goto end;
   }
 
-  player = playerctl_player_new(player_name, &error);
-  if (error != NULL) {
-    g_printerr("Connection to player failed: %s\n", error->message);
-    exit_status = 1;
-    goto end;
-  }
+  if (player_name_list == NULL) {
+    player = playerctl_player_new(player_name_list, &error);
+  } else {
+    gchar *player_names = player_name_list;
+
+    if (g_strcmp0(player_names, "*") == 0) {
```

Comment:
I don't know if I like the use of `*` for this because for one, it's a shell glob so you have to quote it or it expands to all files in the directory, and two it gives the false impression of being a glob so you might think you can do something like `vlc*` and it will select all the vlc instances.